docs/management/index-patterns.asciidoc

```diff
@@ -20,96 +25,103 @@ image::images/management-index-read-only-badge.png[Example of Index Pattern Mana
+An index pattern can match
+only one rollup index.
 
 [float]
-==== Rollup index pattern
+[[management-cross-cluster-search]]
+==== {ccs-cap} index pattern
 
-If a rollup index is detected in the cluster, clicking *Create index pattern* 
-includes an item for creating a rollup index pattern. You create an 
-index pattern for rolled up data the same way you do for any data.  
+If your {es} clusters are configured for {ref}/modules-cross-cluster-search.html[{ccs}], you can create
+index patterns to search across the clusters of your choosing. Using the
+same syntax that you'd use in a raw {ccs} request in {es}, create your
+index pattern with the convention `<cluster-names>:<pattern>`.
 
-You can match an index pattern to only rolled up data, or mix both rolled 
-up and raw data to visualize all data together. An index pattern can match 
-only one rollup index, not multiple. There is no restriction on the 
-number of standard indices that an index pattern can match.
+For example, to query {ls} indices across two {es} clusters
+that you set up for {ccs}, which are named `cluster_one` and `cluster_two`,
+you would use `cluster_one:logstash-*,cluster_two:logstash-*` as your index pattern.
 
-See <<visualize-rollup-data, Creating a visualization using rolled up data>> 
-for more detailed information.
+You can use wildcards in your cluster names
+to match any number of clusters, so if you want to search {ls} indices across
+clusters named `cluster_foo`, `cluster_bar`, and so on, you would use `cluster_*:logstash-*`
+as your index pattern.
+
+To query across all {es} clusters that have been configured for {ccs},
+use a standalone wildcard for your cluster name in your index
+pattern: `*:logstash-*`.
+
+Once an index pattern is configured using the {ccs} syntax, all searches and
+aggregations using that index pattern in {kib} take advantage of {ccs}.
 
 [float]
 === Manage your index pattern
 
-Once you’ve created an index pattern, you’re presented a table of all fields 
-and associated data types in the index. 
+Once you create an index pattern, manually or with a sample data set,
+you might want to look at its fields and associated data types.
+You might also want to perform housekeeping tasks, such as making the
+index pattern the default or deleting it when you longer need it.
+You can drill down into the details of an index pattern by clicking its name in
+the *Index patterns* overview.
 
 [role="screenshot"]
 image:management/index-patterns/images/new-index-pattern.png["Index files and data types"]
 
-You can perform the following actions:
+From this view, you can perform the following actions:
 
-* *Manage the index fields.* Click a column header to sort the table by that column. 
-Use the field dropdown menu to limit to display to a specific field. 
-See <<managing-fields, Managing fields>> for more detailed information. 
+* *Manage the index fields.* Click a column header to sort the table by that column.
```

Comment:
I think this section should drop text about sorting columns and filtering fields (seems irrelevant and most tables in Kibana allow these types of interactions).  Instead, maybe add text about why a user would want to manage fields, such as adding formatters to format values and creating scripted fields.

Reply:
++ Agree. I think the "why" is more important than the "how" in this section.